_posts/2015-05-21-use-docker-to-try-pulp.md

```diff
@@ -12,12 +12,12 @@ categories:
 Using Docker, you can now easily and quickly stand up a demo deployment of Pulp. Excluding the time it takes Docker to download the images, it takes only a few seconds to have a brand new deployment running.
 
     
-    $ wget https://raw.githubusercontent.com/pulp/pulp_packaging/master/dockerfiles/centos/start.sh
+    $ wget https://raw.githubusercontent.com/pulp/packaging/docker/dockerfiles/centos/start.sh

```

Comment:
@bmbouter : Updated. 

I use `vim` with `ffs=unix,dos` in my `.vimrc`.  Still this file shows `^M` character. Strange.